geodiff/src/geodiff.h

```diff
@@ -502,6 +503,19 @@ GEODIFF_EXPORT void GEODIFF_V_getData( GEODIFF_ValueH valueHandle, char *data );
 GEODIFF_EXPORT void GEODIFF_V_destroy( GEODIFF_ValueH valueHandle );
 
 
+/**
+ * Extracts WKB geometry from the geometry encoded according to GeoPackage spec.
+ * Strips GeoPackageBinaryHeader from the StandardGeoPackageBinary envelope and
+ * returns geometry as WKB. More details at https://www.geopackage.org/spec/#gpb_format
+ */
```

Comment:
can you add description of the args + return (like return value + length is buffer you need to free in GEODIFF_free)